README.asciidoc

```diff
@@ -210,7 +209,6 @@ imageBuild {
 |`webadapter` | `Property<String>` | `docker.intershop.de/intershop/icm-webadapter:latest`       | The image contains the web server of the Intershop Commerce Management application with a special Intershop Apache module.
 |`webadapteragent`  | `Property<String>` | `docker.intershop.de/intershop/icm-webadapteragent:latest`       | The web adapter agent image of the Intershop Commerce Management application.
 |`mssqldb`  | `Property<String>` | `mcr.microsoft.com/mssql/server:2019-latest`       | The Microsoft SQL  server image. There will be also a build file available.
```

Comment:
I know you didn't touch this, but the mssql image is incorrect and the solr one should also be `solr:latest`. But now that I look at it most of the images are actually wrong, since they are now all hosted on dockerhub. So maybe we should fix all of them in a separate PR.

Reply:
Yes, maybe we should fix this inside a separate PR.